### DISCUSSION TITLE
Monthly ElectionGuard Office Hours

### DISCUSSION THREAD

**Discussion by rc-ms** [microsoft/electionguard#78] [General]
Office hours are open to the public and occur the first Wednesday of every month. If you have questions about ElectionGuard and want to talk to the core team and community members, please join.Please also feel free to ask questions below in advance, especially if specific technical issues are involved. Thank you and we look forward to seeing you!

Upcoming meetings:

- [March 2 12:00 PM](https://aka.ms/eg-office-hrs) Pacific Standard Time (US)
- [April 6 12:00 PM PST](https://aka.ms/eg-office-hrs)
- [May 4 12:00 PM PST](https://aka.ms/eg-office-hrs)

**Comment by echumley-msft** [comment]
Hi all - we'll be cancelling today's meeting (July 7th) - but please drop any questions or comments here in the meantime

**Comment by ruescasd** [comment]
Hello team,

I am currently evaluating electionguard for use in online voting. Are these meetings open to the general public?

**Reply by rc-ms** [reply]
Hello @ruescasd they are and you're welcome to join. Also feel free to reach out to the team directly with questions about implementation or let us know about any upcoming elections or voting-method support you plan to pursue; we'd love to hear from you. Please contact us at msddp@microsoft.com

**Reply by ruescasd** [reply]
Thank you for the reply and schedule, I may join on the next one.

**Reply by amarealcoder** [reply]
Hello team I would really love to contribute and be a part of this project as developer and technical writer.

**Reply by rc-ms** [reply]
hello @amarealcoder please join our calls but also jump in on documentation issues or anything labeled "good first issue" or "hacktoberfest"

**Comment by amarealcoder** [comment]
Thank you. How do I join the calls?

**Reply by rc-sea** [reply]
Hello! Simply click the link at the top of the page ([reproduced here](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZDc2MmVkOGEtN2E5YS00NGFkLWI2MzgtNmQ0MTdlZmYzOTFj%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22ec3bd50a-8bc6-4662-9553-cf2d3df9f3f4%22%7d)) at the appointed time: [12 pm pacific on Wed Nov 3](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZDc2MmVkOGEtN2E5YS00NGFkLWI2MzgtNmQ0MTdlZmYzOTFj%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22ec3bd50a-8bc6-4662-9553-cf2d3df9f3f4%22%7d)

**Comment by rc-ms** [comment]
Agenda for 11/3

- introduce new members
- summary of hacktoberfest / new contributions
- discuss latest release just pushed
- outline upcoming release
- discuss json schema work
- anything else?
